MediaGallerySynchronization/Model/CreateAssetFromFile.php

```diff
@@ -70,6 +79,7 @@ public function execute(\SplFileInfo $file)
 
         return $this->assetFactory->create(
             [
+                'id' => $this->getAssetId($path),
                 'path' => $this->getPath($path),
```

Comment:
Thanks for suggestion @coderimus 
```suggestion
        $relativePath = $this->getPath($path)

        return $this->assetFactory->create(
            [
                'id' => $this->getAssetId($relativePath),
                'path' => $relativePath,
```